modules/ossm-security-mtls.adoc

```diff
@@ -99,3 +118,19 @@ The default is `TLS_AUTO` and does not specify a version of TLS.
 |`TLSv1_3`
 |TLS version 1.3
 |===
+
+[id="ossm-security-enabling-controlplane_{context}"]
+== Enable strict mTLS in for Mixer telemetry or policy components
+
+Secure connections are always used when proxies communicate with the control plane regardless of `spec.security.controlPlane.mtls` setting. If you are using Mixer telemetry or policy set `spec.security.controlPlane.mtls` to `true` in your `ServiceMeshControlPlane` resource to enable 
```

Comment:
* regardless of -> regardless of the 
* The last sentence doesn't have punctuation or make any sense :)